api/src/main/java/org/pmiops/workbench/access/AccessTierService.java

```diff
@@ -0,0 +1,18 @@
+package org.pmiops.workbench.access;
+
+import java.util.List;
+import org.pmiops.workbench.db.model.DbAccessTier;
+import org.pmiops.workbench.db.model.DbUser;
+
+public interface AccessTierService {
+  String REGISTERED_TIER_SHORT_NAME = "registered";
```

Comment:
nit: this feels  enum-like to me now - is this temporary while we only truly have one access group? Or should this actually be an enum?

Reply:
it supports the privileged nature of Registered in some contexts, which is intended to be temporary.